src/admin-views/category-colors/category-color-preview.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Template for displaying a category color preview in the admin table.
+ *
+ * @since TBD
+ *
+ * @var string $primary   The primary color hex value.
+ * @var string $secondary The secondary color hex value.
+ * @var string $text      The text color hex value.
+ * @var string $priority  The priority of the category.
+ * @var string $hide_from_legend    Whether the category is hidden.
+ */
+
+?>
+<span class="tec-events-taxonomy-table__category-color-preview"
+	style="background-color: <?php echo esc_attr( $secondary ); ?>; border: 3px solid <?php echo esc_attr( $primary ); ?>;"
```

Comment:
Is there a way we can do this via classes (like, the actual classes used on the front end perhaps) and custom props?
I dislike inline styles, even in the admin.